tests/gha-run-k8s-common.sh

```diff
@@ -350,6 +370,7 @@ function deploy_k8s() {
 		k0s) deploy_k0s ;;
 		k3s) deploy_k3s ;;
 		rke2) deploy_rke2 ;;
+		microk8s) deploy_microk8s ;;
```

Comment:
We are currently not running those tests but I'd say to leave this commit in, as once I have some time to revisit the tests I'll appreciate that we have some sort of code related to using microk8s.

Reply:
Nops, that's on me to bring the tests up again, so you did more, way more, than expected. :-)